src/NumericalAlgorithms/Interpolation/PredictedZeroCrossing.hpp

```diff
@@ -16,4 +18,16 @@ namespace intrp {
 double predicted_zero_crossing_value(const std::vector<double>& x_values,
                                      const std::vector<double>& y_values);
 
+/*!
+ * \brief Predicts the zero crossing of multiple functions contained in a vector
+ * of datavectors.
+ *
+ * Fits a set of N linear functions, where N is the size of the returned
```

Comment:
In the documentation comment, please add a longer description of what is expected in x_values and y_values.  In particular, please specify the indexing order of y_values, please specify which sizes in x_values and y_values are supposed to be the same, and please specify the size of the returned value.